addons/calendar/models/calendar_event.py

```diff
@@ -222,6 +225,23 @@ def _compute_is_highlighted(self):
             for event in self:
                 event.is_highlighted = False
 
+    @api.depends('partner_id', 'attendee_ids')
+    def _compute_is_organizer_alone(self):
+        """
+            Check if the organizer of the event is the only one who has accepted the event.
+            It does not apply if the organizer is the only attendee of the event because it
+            would represent a personnal event.
+            The goal of this field is to highlight to the user that the others attendees are
+            not available for this event.
+        """
+        for event in self:
+            organizer = event.attendee_ids.filtered(lambda a: a.partner_id == event.partner_id)
+            all_declined = not any((event.attendee_ids - organizer).filtered(lambda a: a.state != 'declined'))
```

Comment:
I don't understand this computation. It seems you want an alarm if "all participants have declined except organizer". You should filter out organized from attendees than compute field based on it. Not sure why there is a "greater than 1 ".